Documentation/Developer.md

```diff
@@ -36,7 +36,7 @@ From the top level data-transfer-project directory:
 * Install Git: `sudo apt-get install git-all`
 * Install Gradle: `sudo apt-get install gradle`
  * Install NVM: `curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash`
- * Install Node: `nvm install node`
+ * Install Node: `nvm install v10.16.3`
```

Comment:
I'm actually not 100% sure this command does what it's supposed to, I haven't actually run it on linux yet

Reply:
Confirmed that this works